ember.md

```diff
@@ -98,6 +100,49 @@ get(this, 'isDestructured'); //true
 set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
+
+<a name="data"></a>
+## Data
+
+<a name="data--data-binding"></a>
+### 3.1 Data Binding
+
+Any new code written by ASH should only contain one-way data-binding. Third-party addons using two-way data-binding is ok, but be cautious and conscious of the side effects it can have.
+
+> Why? Two-way data-binding can have unexpected side effects; data flowing one way keeps things more predictable. For example: if you pass the same data to 2 components and they both are allowed to change it. DDAU allows the parent to arbitrate those changes and resolve conflicts. Otherwise component A can make changes you are not expecting in component B. This is not just a matter of 2 compoents, you can make a change to a child that propagates through the whole app.
+
+```javascript
+//Bad
+{{my-cheesy-component 
+  myData=parentData
+}}
+
+//my-cheesy-component.js
+//The below code would update the parent and any other component the parentData object was passed to
+set(myData, 'cheese', 'gouda')
+
+
+//Good
+{{my-cheesy-component 
+  myData = parentData
+  changeCheese = (action 'newCheesePlease')
+}}
+
+//my-cheesy-component.js
+actions: {
+  quesoChanger(cheeseType) {
```

Comment:
for consistency with the bad example, i would remove `cheeseType` and just hardcode `'gouda'` on line 134 to show where `gouda` comes from